react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('userState', isClicked);
+
+  function handleClick() {
+    console.log(isClicked);
+    setIsClicked(true);
```

Comment:
By setting it to true, we can never set this back to false. This should be `setIsClicked(!isClicked)`